Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+---
+title: "Use IEdmTargetPath for out-of-line annotations target"
+description: "Use IEdmTargetPath for out-of-line annotations target"
+author: kenitoinc
+ms.author: kemunga
+ms.date: 05/2/2023
+ms.topic: article
+ 
+---
+# Use IEdmTargetPath for out-of-line annotations target
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../../includes/appliesto-odatalib-v7.md)]
+
+The Edm library lets you [define annotations](/odata/odatalib/edm/define-annotations).
+
+[This](/odata/odatalib/edm/set-annotations-schema) documentation describes how to define **out-of-line** annotations that targets a model element.
+
+## Background
+
+Below is a simple Edm model with an out-of-line annotation that targets a model element.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+    <edmx:DataServices>
+        <!-- Model elements -->
+        <Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <EntityContainer Name="Container">
+                <EntitySet Name="Employees" EntityType="Ns.Employee" />
+            </EntityContainer>
+            <Annotations Target="Ns.Employee">
+                <Annotation Term="Ns.MyTerm" String="Name OutOfLine MyTerm Value" />
+            </Annotations>
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>
+```
+
+`IEdmTargetPath` allows us to define out-of-line annotations that target a path to a model element.
+
+Below are examples of paths to a model element.
+```
+MySchema.MyEntityContainer/MyEntitySet/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MyNavigationProperty
+MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyNavProperty
+MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyProperty
+MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyNavigationProperty
+MySchema.MyEntityContainer/MySingleton/MyComplexProperty/MyNavigationProperty
+```
+
+Below is a simple Edm model with an out-of-line annotation that targets a path to a model element.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+    <edmx:DataServices>
+        <!-- Model elements -->
+        <Schema Namespace="Default" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+            <EntityContainer Name="Container">
+                <EntitySet Name="Employees" EntityType="Ns.Employee" />
+            </EntityContainer>
+            <Annotations Target="Default.Container/Employees/Ns.Manager/EmployeeId">
+                <Annotation Term="Ns.MyTerm" String="Name OutOfLine MyTerm Value" />
+            </Annotations>
+        </Schema>
+    </edmx:DataServices>
+</edmx:Edmx>
+```
```

Comment:
How is this different from the earlier sample and what new value does it add?
I think you should get rid of it...
Instead you should introduce the following sections where you're going to describe how to use your feature to set the target
```suggestion
In the sections that follow, we cover how to set the target for an out-of-line annotation using the Edm model builder.
```